src/python/turicreate/test/test_object_detector.py

```diff
@@ -285,6 +285,13 @@ def test_evaluate_invalid_threshold(self):
         with self.assertRaises(_ToolkitError):
             self.model.evaluate(self.sf.head(), iou_threshold  =-1)
 
+    def test_evaluate_sframe_format(self):
+        metrics = ["mean_average_precision_50","mean_average_precision"]
+        for metric in metrics:
+            pred = self.model.evaluate(self.sf.head(), metric=metric, output_type="sframe")
+            self.assertEqual(len(pred.column_names()), 1)
+            self.assertEqual(pred.column_names()[0], "label")
```

Comment:
This looks wrong to me, but I think it's alright. You can't count on the column ordering of SFrames to always be the same, but I think this is ok cause there is only one column. In which case I think it would be better to do:
`self.assertEqual(pred.column_names(), ["label"])

Reply:
good!